README.rst

```diff
@@ -65,13 +69,15 @@ Usage Example
 
     import time
     import board
-    import adafruit_tmp117
+    import adafruit_tmp11X
+    from adafruit_tmp11X import TMP117, TMP116, TMP119, AverageCount, MeasurementMode, MeasurementDelay
 
     i2c = board.I2C()  # uses board.SCL and board.SDA
-    tmp117 = adafruit_tmp117.TMP117(i2c)
-
+    t1 = TMP116(i2c_bus=i2c, address=0x49)
+    # t2 = TMP117(i2c_bus=i2c, address=0x49)
+    # t2 = TMP119(i2c_bus=i2c, address=0x49)
```

Comment:
Are the I2C addresses all the same? If so don't bother to specify the address here, since the default is fine and makes usage easier.

Reply:
It appears the default I2C address is the same. It's just that it can be varied via pin settings.